### DISCUSSION TITLE
Share your showcase with us!

### DISCUSSION THREAD

**Discussion by hardfist** [web-infra-dev/rspack#3170] [Show and tell]
Hello everyone 🙌,
**Do you use Rspack in production or for hobby project?**  Please share your website or project with us by leaving comment below. Don't forget to include a link to your website or project and a brief description of what you have created.
Some examples:
* apaas: https://apaas.feishu.cn/
* Rspress: https://rspress.dev/
* 抖音集星: https://www.lifecreator.cn/

**Comment by chenjiahan** [comment]
https://discord.com/ is migrating from Webpack to Rspack and production build times are down over 70% 📉

![image](https://github.com/web-infra-dev/rspack/assets/7237365/5d7b1314-5df4-493b-93ee-0f31e67579ea)

> ref: https://twitter.com/aweary/status/1716890502682018290

**Reply by nailfar** [reply]
快是快了，没有物理缓存都在内存缓存，大型项目真的扛不住，几个热更新内存就崩溃了

**Comment by chenjiahan** [comment]
[Why you should migrate to Rspack from webpack](https://blog.logrocket.com/migrate-rspack-webpack/) by @LogRocket

> from: https://twitter.com/LogRocket/status/1716462484062601543

**Comment by chenjiahan** [comment]
10x faster [@猿辅导](https://www.yuanfudao.com/)

![image](https://github.com/web-infra-dev/rspack/assets/7237365/98f9d206-6677-4769-ae7b-443768501f9e)

> from: https://weibo.com/1649383554/4959969842566134

**Comment by chenjiahan** [comment]
Rspack has been integrated into ice.js, Alibaba's (阿里巴巴) universal framework.

> from: https://github.com/alibaba/ice/pull/6440

**Reply by hardfist** [reply]
more https://github.com/ice-lab/icepack

**Comment by cunzaizhuyi** [comment]
https://twitter.com/sumygg/status/1719993104592425311
小米

**Comment by hardfist** [comment]
7x faster than Vite in Baidu Vue Project
![image](https://github.com/web-infra-dev/rspack/assets/8898718/c739d224-7ba1-4601-97a7-9f21a04e321a)

https://twitter.com/__ty96/status/1722187939156021556

**Comment by antoinepairet** [comment]
At [Rosa](https://rosa.be), we have been using Rspack in production for 9 months. We are a team of 20+ engineers.
We have seen mind-blowing speed increase and the ability to bundle all dependencies (avoiding the need for `npm i`):

- Rpsack is between 5x and 10x faster than Webpack
- By bundling the dependencies we have:
    - Decreased the size of our Docker images
    - Went from a 10+ min build to <5s for the serverless framework

Here are the slides of I talk I gave about it: https://docs.google.com/presentation/d/1nmN8Nruv_hshSbmhRWP1ikOIpmzAPL5xcpzyc-mOLNc/edit#slide=id.g2412ef3412a_0_321

**Comment by cunzaizhuyi** [comment]
[sentry](https://twitter.com/yagiznizipli/status/1734336798615027715)
webpack=> rspack 
58s => 14s

**Comment by abenhamdine** [comment]
at @arhia we used to use vite to build our react app, but we frequently ran into OOM and were unable to update ant-design because of that
Now with rspack, no more OOM and we now are on antd last version.
So congrats 🎉

**Comment by 1yasa** [comment]
[IF：Typora free alternative](https://apps.apple.com/us/app/if-gtd-for-professionals/id6460080040?mt=12)

We have greatly accelerated the application iteration speed using rspack.

**Comment by chenjiahan** [comment]
- DeWu （得物）：https://juejin.cn/post/7389925302020014118
- KuaiShou（快手）：https://juejin.cn/post/7425804396292325414
